hack/aws/README.md

```diff
@@ -0,0 +1,10 @@
+# AWS Helpers
+
+### supported_types
+Used to check every Availability Zone to see that specific instance types are supported.
+
+We should check for both current instance types used by CRO and their legacy equivalent. 
```

Comment:
The user needs to ensure they have the `aws` CLI setup and configured against the account they want to check.

Maybe we should also include a link to [1] so they can understand what that availability zone is related to e.g. eu-north-1 means Stockholm. 

Also, might be good to note that this will only include opt-in regions like South Africa if the AWS account has already opted into them.

[1] https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions